types/misc.go

```diff
@@ -0,0 +1,3 @@
+package types
```

Comment:
can we name this file as constants.go and then can add more constants to single file

Reply:
done. I think at some point we should move it to a constants pkg maybe?
const.EigenPromNamespace is more explicit than types.EigenPromNamespace.
Will leave this for a future PR though. Maybe move `types/` and `consts/` under `common/` or something?

Reply:
Actually don't people use an `internal/` package for things that shouldn't be exposed? I feel like this eigen namespace shouldn't be exposed, so maybe put it there?

Reply:
Yea we could have something like internal if we don't have to expose it. But should be expose? If AVs wants to use that in any way?